Xamarin.Forms.Core/Slider.cs

```diff
@@ -6,25 +6,25 @@ namespace Xamarin.Forms
 	[RenderWith(typeof(_SliderRenderer))]
 	public class Slider : View, IElementConfiguration<Slider>
 	{
-		public static readonly BindableProperty MinimumProperty = BindableProperty.Create("Minimum", typeof(double), typeof(Slider), 0d, validateValue: (bindable, value) =>
+		public static readonly BindableProperty MaximumProperty = BindableProperty.Create("Maximum", typeof(double), typeof(Slider), 1d, validateValue: (bindable, value) =>
 		{
 			var slider = (Slider)bindable;
-			return (double)value < slider.Maximum;
+			return (double)value > slider.Minimum;
 		}, coerceValue: (bindable, value) =>
 		{
 			var slider = (Slider)bindable;
-			slider.Value = slider.Value.Clamp((double)value, slider.Maximum);
+			slider.Value = slider.Value.Clamp(slider.Minimum, (double)value);
 			return value;
 		});
 
-		public static readonly BindableProperty MaximumProperty = BindableProperty.Create("Maximum", typeof(double), typeof(Slider), 1d, validateValue: (bindable, value) =>
+		public static readonly BindableProperty MinimumProperty = BindableProperty.Create("Minimum", typeof(double), typeof(Slider), 0d, validateValue: (bindable, value) =>
 		{
 			var slider = (Slider)bindable;
-			return (double)value > slider.Minimum;
+			return (double)value < slider.Maximum;
 		}, coerceValue: (bindable, value) =>
 		{
 			var slider = (Slider)bindable;
-			slider.Value = slider.Value.Clamp(slider.Minimum, (double)value);
+			slider.Value = slider.Value.Clamp((double)value, slider.Maximum);
```

Comment:
The reason for this shuffling was to organize Stepper and Slider bindable properties in the same way. I was comparing validate and coerce functions and it was easier if the order was the same.